frontend/src/store/todos-context.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import TodosContextProvider, { TodosContext } from './todos-context';
+
+describe('TodoProvider', ()=>
+{
+	test('check length !== 0', ()=>
+	{
+
+		render(<TodosContextProvider>
+			<TodosContext.Consumer>
+				{
+					value => <span>Is length: {value.fetchedItems}</span>
+				}
+			</TodosContext.Consumer>
+		</TodosContextProvider>);
+
+	expect(('Is length')).not.toHaveLength(0);
+
+	});
+	test('check length === exact number', ()=>
+	{
+
+		render(<TodosContextProvider>
+			<TodosContext.Consumer>
+				{
+					value => <span>Is length: {value.fetchedItems}</span>
+				}
+			</TodosContext.Consumer>
+		</TodosContextProvider>);
+
+	expect(('is length')).not.toHaveLength(0);
+
+	});
+	test('check add todo func', ()=>
```

Comment:
always a empty line between functions - this is an example, please check the whole project for that as I see this everywhere from time to time